Can not compile IFS-FESOM on MN5 anymore

When I try to build IFS-FESOM with fesom 2.6.1 instead of 2.5.06_de on MN5, I do get:

```
make[2]: Circular fesom/src/CMakeFiles/fesom.dir/gen_modules_clock.F90.o <- fesom/src/CMakeFiles/fesom.dir/mod_parsup.mod.stamp dependency dropped.
[ 41%] Building Fortran object fesom/src/CMakeFiles/fesom.dir/gen_modules_clock.F90.o
/gpfs/projects/ehpc01/sbeyer/models/DE_CY48R1.0_climateDT_tco399/source/fesom/src/gen_modules_clock.F90(72): error #7002: Error in opening the compiled module file.  Check INCLUDE paths.   [MOD_PARSUP]
    USE MOD_PARSUP
--------^
compilation aborted for /gpfs/projects/ehpc01/sbeyer/models/DE_CY48R1.0_climateDT_tco399/source/fesom/src/gen_modules_clock.F90 (code 1)
make[2]: *** [fesom/src/CMakeFiles/fesom.dir/build.make:556: fesom/src/CMakeFiles/fesom.dir/gen_modules_clock.F90.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:34704: fesom/src/CMakeFiles/fesom.dir/all] Error 2
make: *** [Makefile:166: all] Error 2
```

On levante it worked just fine.

https://github.com/FESOM/fesom2/compare/2.5.0.6_de...2.6.1

any ideas why this happens?

--- a/src/gen_modules_clock.F90
+++ b/src/gen_modules_clock.F90
@@ -69,7 +69,6 @@ contains
   !
   subroutine clock_init(partit)
     USE MOD_PARTIT
-    USE MOD_PARSUP
     use g_config
     use mod_transit, only: ti_transit, ti_start_transit
     implicit none

